src/components/Accounts/Transactions/Transaction/index.js

```diff
@@ -3,6 +3,28 @@ import moment from 'moment';
 import CategoryIcon from 'components/CategoryIcon';
 
 export default class Transaction extends React.Component {
+
+
+  static getTextualDecline(declinedReason){
+	if(!declinedReason) return false;
```

Comment:
Are you talking about comment blocks? If so I've added one to the function, if not I'm not sure what you mean.